uber/automated_emails.py

```diff
@@ -213,7 +213,7 @@ def days_before(days, dt, until=None):
                  and state.BEFORE_PREREG_TAKEDOWN and a.takes_shifts and not a.hours)
 
 StopsEmail('Last chance to sign up for {EVENT_NAME} shifts', 'shifts/reminder.txt',
```

Comment:
This is unrelated but I noticed was failing because of a typo.  It was also generated hundreds of log messages every 5 minutes :(